build-android/build_all.sh

```diff
@@ -51,10 +51,10 @@ echo LAYER_BUILD_DIR="${LAYER_BUILD_DIR}"
 echo CUBE_BUILD_DIR="${CUBE_BUILD_DIR}"
 
 function create_APK() {
-    aapt package -f -M AndroidManifest.xml -I "$ANDROID_SDK_HOME/platforms/android-23/android.jar" -S res -F bin/$1-unaligned.apk bin/libs
```

Comment:
I updated this to match what is in `Application.mk` so that only platform-26 is needed to build (not 26 _and_ 23).